decidim-admin/app/models/decidim/admin/concerns/user_extends.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+require "active_support/concern"
+
+module Decidim
+  module Admin
+    # A module that adds functionalities to the Decidim::User model class.
+    module UserExtends
+      extend ActiveSupport::Concern
+
+      included do
+        has_many :participatory_process_user_roles, foreign_key: "decidim_user_id", class_name: Decidim::ParticipatoryProcessUserRole
+        has_many :participatory_processes, through: :participatory_process_user_roles
```

Comment:
@oriolgual thoughts? I don't want to start changing this and then having to undo all changes, since this rewrite is not a 4 lines change :/